### PR TITLE
adds a new mutation calcification exclusively found in skeletons

### PR DESCRIPTION
**Pull request by boodaliboo**
just like fire breath this mutation is only found in skeletons but can be given to anyone once found
infact thats the main point 
this mutation gives any non skeleton skeletons enhanced healing from calcium
#### Changelog

:cl:  
rscadd: Added a new mutation named calcification
/:cl:

### CODE REVIEW

File: code/__DEFINES/DNA.dm [hunk 1]
```diff
@@ -50,6 +50,8 @@
 #define EXTRASTUN	/datum/mutation/human/extrastun
 #define GELADIKINESIS		/datum/mutation/human/geladikinesis
 #define CRYOKINESIS /datum/mutation/human/cryokinesis
+#define Calcification	/datum/mutation/human/calcification
+
```
Comment (JamieD1) [inline code/__DEFINES/DNA.dm:54]: Spaces

**Review by JamieD1 — APPROVED** [review]

**Comment by Time-Green** [comment]
This works? Skeletons don't have dna and are rad immune

**Comment by alexkar598** [comment]
They dont have DNA....

**Comment by Time-Green** [comment]
bruh moment

**Pull request closed without merging**